rabix-bindings-cwl/src/main/java/org/rabix/bindings/cwl/CWLBindings.java

```diff
@@ -208,4 +208,19 @@ public Object transformInputs(Object value, Job job, Object transform) throws Bi
   public DataType getDataTypeFromValue(Object input) {
     return CWLSchemaHelper.getDataTypeFromValue(input);
   }
+
+  @Override
+  public List<FileValue> getFilesFromValue(Object input) {
+    return CWLSchemaHelper.getFilesFromValue(input);
+  }
+
+  @Override
+  public Object updateFileValues(Object input, Map<FileValue, FileValue> files) {
+    return CWLSchemaHelper.updateFileValues(input, files);
+  }
+
+  @Override
+  public Object updateFilePaths(Object input, Map<String, String> paths) {
```

Comment:
This functionality is similar as the one above. Method is updateFileValues(Object input, FileTransformer)